### DISCUSSION TITLE
Lost the buttons column?

### DISCUSSION THREAD

**Discussion by PaperOrb** [Kryptortio/SDAtom-WebUi-us#34] [Q&A] [answered]
Maybe I missed it, but I didn't see any issues onw how to retrieve a column in the task queue if you accidentally drag it off to the right. I used the arrow keys to move to the far right column, but I still didn't see the play and delete buttons column

**Comment by Kryptortio** [comment, marked as answer]
Don't know what you mean, you can't normally drag any of the elements this script adds. Maybe you are using some other kind of extension? There is also no play column although I guess ❌ buttons could be called a delete column...

**Reply by PaperOrb** [reply]
Apologies, I had this extension mixed up with agent scheduler.
